### PR TITLE
(DOCSP-22987): Release notes, MongoDB World 2022

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-22987

### Staged Changes (Requires MongoDB Corp SSO)

- [Release Notes](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-22987/release-notes/backend/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released Data API as GA.
+- Introduced field-level permissions for Flexible Sync.
+- Released Asymmetric Sync, which optimizes Flexible Sync for write-heavy workloads.
+- Introduced Data API for any App.
+- Introduced local regions for Azure data sources.
+- Introduced Atlas Alerts for failed requests that have met a certain threshold.
```
Comment (mongodben) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:12]: i find the wording here kinda awkward and opaque. 

- what type of requests? 
- is the threshold user-defined? 

could this be rephrased?

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 2]
```diff
@@ -1,7 +1,20 @@
+.. _backend_20220607:
+
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released Data API as GA.
+- Introduced field-level permissions for Flexible Sync.
+- Released Asymmetric Sync, which optimizes Flexible Sync for write-heavy workloads.
+- Introduced Data API for any App.
```
Comment (mongodben) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:10]:
```suggestion
- Introduced Data API for all App.
```

nit

**Review by mongodben — APPROVED** [review]
lgtm. see nitty notes

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst (hunk 2 above)
Comment (nathan-contino-mongo) [reply source/release-notes/changelogs/backend/changelog-backend-2022.rst:10]:
```suggestion
- Introduced Data API for all Apps.
```

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/release-notes/changelogs/backend/changelog-backend-2022.rst:12]:
```suggestion
```

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 3]
```diff
@@ -1,7 +1,19 @@
+.. _backend_20220607:
+
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released Data API as GA.
```
Comment (nlarew) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:7]:
```suggestion
- Released :ref:`Data API <data-api>` as GA.
```

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 4]
```diff
@@ -1,7 +1,19 @@
+.. _backend_20220607:
+
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released :ref:`Data API <data-api>` as GA.
+- Introduced field-level permissions for Flexible Sync.
+- Released Asymmetric Sync, which optimizes Flexible Sync for write-heavy workloads.
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:9]:
```suggestion
- Released :ref:`Asymmetric Sync <optimize-asymmetric-sync>`, which optimizes Flexible Sync for write-heavy workloads.
```

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 5]
```diff
@@ -1,7 +1,19 @@
...
+
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released :ref:`Data API <data-api>` as GA.
+- Introduced field-level permissions for Flexible Sync.
+- Released :ref:`Asymmetric Sync <optimize-asymmetric-sync>`, which optimizes Flexible Sync for write-heavy workloads.
+- Introduced Data API for all Apps.
+- Introduced local regions for Azure data sources.
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:11]:
```suggestion
- Introduced :ref:`local regions for Azure data sources <realm-regions>`.
```

File: source/release-notes/changelogs/backend/changelog-backend-2022.rst [hunk 6]
```diff
@@ -1,7 +1,19 @@
...
+
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released :ref:`Data API <data-api>` as GA.
+- Introduced field-level permissions for Flexible Sync.
+- Released :ref:`Asymmetric Sync <optimize-asymmetric-sync>`, which optimizes Flexible Sync for write-heavy workloads.
+- Introduced Data API for all Apps.
+- Introduced :ref:`local regions for Azure data sources <realm-regions>`.
```
Comment (nathan-contino-mongo) [inline source/release-notes/changelogs/backend/changelog-backend-2022.rst:11]:
```suggestion
- Released Flexible Sync as GA.
- Released :ref:`Asymmetric Sync <optimize-asymmetric-sync>`, which optimizes Flexible Sync for write-heavy workloads.
- Released :ref:`Data API <data-api>` as GA.
- Introduced Data API for all Apps.
- Introduced field-level permissions for Flexible Sync.
- Introduced :ref:`local regions for Azure data sources <realm-regions>`.
```

**Pull request merged**